README.md

```diff
@@ -96,37 +96,46 @@ without the `LIQUID_DEV_MODE`.
 
 ## How To Run Regression Tests
 
+For documentation on the `test-driver` executable itself, please refer to the
+`README.md` in `tests/` or run `cabal run tests:test-driver -- --help` or `stack
+run test-driver -- --help`
+
 _For a way of running the test suite for multiple GHC versions, consult the General Development FAQs. below_
 
 There are particular scripts for running LH in the different modes, e.g. for different 
-compiler versions and in plugin mode or as standalone. These scripts are in:
+compiler versions. These scripts are in:
 
     $ ./scripts/test
 
 So you can run *all* the tests for say the ghc-8.10 version by
 
     $ ./scripts/test/test_810.sh
```

Comment:
This script is being removed in this PR.

Reply:
Changed to existing script.